Fixed some issues in the ESPhome firmware of MR60BHA2

lewuq opened the pull request:
1. Removed the content that would lead to failure.
2. Added some contents related to Wi-Fi and OTA to avoid Wi-Fi connection failure.
3. Added some contents related to API.
4. Added some comments to aid understanding.
5. Removed the "rmt_channel" content from the light section, as it is not present in the library.
6. Used the recommended content from the ESPhome documentation for binary_sensor and sensor sections, excluding "fall_detect".

lewuq committed:
Fixed some issues in the ESPhome firmware of MR60BHA2

limengdu commented:
/translate all

limengdu reviewed (changes requested):
代码版本使用3.5再跑一遍，非必要不要删除代码

actions-user committed:
🌍 自动翻译文档

Translation Languages: zh-CN ja es
Before Translation SHA: 0e3f859366601e4059a624d309bcd48e971551d8
Triggered by: @limengdu
Time: 2025-09-05 03:35:17
